templates/template_flow.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+from metaflow import (
+    FlowSpec, IncludeFile, Parameter,
+    card, current, step,
+    environment, kubernetes
+)
+from metaflow.cards import Markdown
+
+from artifact_store import ArtifactStore
+
+GCS_PROJECT_NAME = "project-name-here"
+GCS_BUCKET_NAME = "bucket-name-here"
+
+class TemplateFlow(FlowSpec):
```

Comment:
@chelseatroy would you kindly file a ticket for wiring up CI for this repo? I think at the very least we want:

* Linting (metaflow linting!)
* An action to make sure that the flow runs (without submitting it to OB)